src/assets/locales/de/main.json

```
@@ -2177,7 +2177,8 @@
     },
     "userRoles": {
       "title": "Assigned Catena-X Portal Roles",
-      "changeRoleBtn": "Change Portal Role"
+      "changeRoleBtn": "Change Portal Role",
+      "errorMsg": "Sie sind nicht berechtigt, Ihre eigenen Rollen zu ändern. Bitte wenden Sie sich an einen anderen Unternehmens- oder IT-Administrator."
```

Descriptions in de and en should be consistent:
```suggestion
      "errorMsg": "Sie sind nicht berechtigt, Ihre eigenen Administrator-Rollen zu ändern. Bitte wenden Sie sich an einen anderen Unternehmens- oder IT-Administrator."
```

---

Done